tests/domain/test_grid.py

```
@@ -0,0 +1,69 @@
+import pytest
+
+from src.domain.entities.grid import HexGrid, InvalidGridPosition
```

Ruff (0.8.2)

`src.domain.entities.grid.InvalidGridPosition` imported but unused

Remove unused import: `src.domain.entities.grid.InvalidGridPosition`

(F401)